extensions/tmtv/src/getHangingProtocolModule.js

```diff
@@ -16,37 +16,87 @@ const ptCT = {
   imageLoadStrategy: 'interleaveTopToBottom', // "default" , "interleaveTopToBottom",  "interleaveCenter"
   protocolMatchingRules: [
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'ModalitiesInStudy',
       constraint: {
         contains: ['CT', 'PT'],
       },
-      required: false,
     },
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'StudyDescription',
       constraint: {
-        contains: {
-          value: 'PETCT',
-        },
+        contains: 'PETCT',
       },
-      required: false,
     },
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'StudyDescription',
       constraint: {
-        contains: {
-          value: 'PET/CT',
-        },
+        contains: 'PET/CT',
       },
-      required: false,
     },
   ],
+  displaySetSelectors: {
+    ctDisplaySet: {
+      seriesMatchingRules: [
+        {
+          weight: 1,
+          attribute: 'Modality',
+          constraint: {
+            equals: {
+              value: 'CT',
+            },
+          },
+          required: true,
+        },
+        {
+          attribute: 'SeriesNumber',
+          constraint: {
+            equals: '4',
+          },
+        },
```

Comment:
I wanted to remove this, but can you on this PR?

Reply:
Removed.